codepy/toolchain.py

```diff
@@ -368,23 +368,16 @@ class ToolchainGuessError(Exception):
 
 
 def _guess_toolchain_kwargs_from_python_config():
-    def strip_prefix(pfx, value):
-        if value.startswith(pfx):
-            return value[len(pfx):]
-        else:
-            return value
-
-    from distutils.sysconfig import parse_makefile, get_makefile_filename
-    make_vars = parse_makefile(get_makefile_filename())
-
-    cc_cmdline = (make_vars["CXX"].split()
-            + make_vars["CFLAGS"].split()
-            + make_vars["CFLAGSFORSHARED"].split())
-    object_names = [
-            oname for oname in make_vars["OBJECT_OBJS"].split()
-            if "(" not in oname and ")" not in oname]
-
-    object_suffix = ".{}".format(object_names[0].split(".")[1])
```

Comment:
This just tries to get the extension for objects on the system, right?

It seems `sysconfig` has a dumber `parse_makefile` that doesn't parse multiline variables. This switched to check `MODOBJS`, which seems to be the only variable in the makefile that's defined on a single line :\